app/move/views/view.njk

```diff
@@ -204,6 +204,28 @@
   {% endif %}
 {% endmacro %}
 
+{% macro renderAssessmentComponent(assessment) %}
```

Comment:
The macros got a lot going on it - could it be handled better by dealing wrapping the differing content in a macro call?

```njk
{% macro renderAssessmentComponent(assessment) %}
  {% if assessment.count > 0 %}
    {% if caller %}{{ caller() }}{% endif %}
    {{ appMessage({
      classes: "app-message--muted govuk-!-margin-top-2",
      allowDismiss: false,
      content: {
        text: t('assessment::no_items.text', {
          context: assessment.key
        })
      }
    }) }}
  {% endif %}
{% endmacro %}
```

which would allow 

```njk
{% call renderAssessmentComponent(assessment) -%}
  {% for panel in assessment.panels %}
    {{ appPanel(panel) }}
  {% endfor %}
{%- endcall %}
...
{% call renderAssessmentComponent(assessment) -%}
  {{ govukSummaryList(assessment) }}
{%- endcall %}
```